backend/internal/storage/table.go

```diff
@@ -0,0 +1,11 @@
+package storage
+
+import "gorm.io/gorm"
+
+// Process model struct for database table
+type Process struct {
+	gorm.Model
+	ProcessID    uint   `gorm:"unique;not null"`
+	ProcessKey   uint   `gorm:"unique;not null"`
```

Comment:
The incoming types from Kafka will be `int64`, so these should be too to fit those values.